CONTRIBUTING.md

```diff
@@ -120,8 +122,7 @@ visibility on the problem and work toward resolution.
 
 ## Closing issues
 
-You can either close issues manually by adding the fixing commit SHA1 to the issue
-comments or by adding the `Fixes` keyword to your commit message:
+You must add a `Fixes` keyword to one of your commits:
```

Comment:
Consider changing it to:
Start the first line of at least one commit message with 'Fix'.
My issue with your version is that the example below does not have the word Fixes but Fix in it.
It is also important to state where in the commit message you are expecting the keyword.